### DISCUSSION TITLE
Adding Channel Number to "Roku" Channels and Guide View.

### DISCUSSION THREAD

**Discussion by megahz2** [jellyfin-archive/jellyfin-roku-legacy#1831] [General]
low priority please "add the channel number to the "live tv" "guide" in Roku?"

**Comment by cewert** [comment]
For feature requests, please fill out an [Enhancement Request](https://github.com/jellyfin/jellyfin-roku/issues/new?assignees=&labels=enhancement&projects=&template=enhancement-request.md&title=). (Issues -> New Issue -> Enhancement Request)
